docs/walk-through/custom-template-variable-reference.md

```diff
@@ -35,5 +35,5 @@ spec:
       container:
         image: busybox
         command: [echo]
-        args: ["{{user.username}}"]
+        args: ["{{message.value}} not working, got value: {{inputs.parameters.message}}"]
```

Comment:
Why do you suggest this has `{{message.value}}` in the example, I'm not following the reason to update the documentation in this way. I understand the `{{inputs.parameters.message}}` part of this change